src/Control/PaginationControl.php

```diff
@@ -496,6 +497,14 @@ protected function createPageSelectorItem()
     protected function assemble()
     {
         if ($this->getPageCount() < 2) {
+            if ($this->getCurrentPageNumber() > 1) {
+                $this->addHtml(new Link(
```

Comment:
Since the pagination numbers are blue (represent action links), I would suggest doing the same here. To do this, please use the class `ActionLink` instead of `Link`.

```php
$this->addHtml(new ActionLink(
    $this->translate('Invalid page, click here to reset'),
    $this->url->without('page'),
    null, //  you can even specify a suitable symbol
    ['title' => $this->createLabel(1)]
));
```